repos/system_upgrade/el8toel9/actors/opensshdropindirectorycheck/actor.py

```diff
@@ -0,0 +1,64 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.rpms import has_package
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM, OpenSshConfig, Report
+from leapp.tags import CheckPhaseTag, IPUWorkflowTag
+
+
+class OpenSshDropInDirectoryCheck(Actor):
+    """
+    Trigger a notice that the main sshd_config will be updated to contain
+    the Include directive so the other configuration files dropped by the
+    RHEL9 packages are effective.
+
+    This might change the sshd behavior so it is advised to verify by the
+    customer that the updated system behaves as expected.
+    """
+
+    name = 'open_ssh_drop_in_directory_check'
+    consumes = (OpenSshConfig, InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (IPUWorkflowTag, CheckPhaseTag,)
```

Comment:
```suggestion
    tags = (IPUWorkflowTag, ChecksPhaseTag,)
```
It is `Checks`, otherwise it would blow up :)

Reply:
Changed also in the import line